qlib/Mapper.qm

```diff
@@ -947,28 +969,29 @@ Mapper mapv(DataMap);
             if (exists m.constant)
                 v = m.constant;
             else if (exists m.index) {
-                v = do_list ? (map m.index + count + $#, v) : m.index + count;
+                v = do_list ? (map m.index + count + $#, v) : m.index + count; # FIXME iterating empty list
             }
             else if (exists m.runtime) {
                 any rtv = m_runtime{m.runtime};
-                v = do_list ? (map rtv, v) : rtv;
+                v = do_list ? (map rtv, v) : rtv; # FIXME iterating empty list
             }
             else if (m.struct) {
+                # NOTE: hash of lists not supported with struct # FIXME do we ensure it?
```

Comment:
this comment needs to be removed - we already established in another already-merged pull request that hash of lists format is supported with `struct` and there are tests for it